### DISCUSSION TITLE
Collection without name

### DISCUSSION THREAD

**Discussion by olegans1972** [seaweedfs/seaweedfs#6208] [General]
When I do the command collection.list  I see a collection without name. In fact,  I didn't create this no-name collection. Could you explain what is the purpose of this collection?
Where does it come from?

> collection.list
collection:"cache"      volumeCount:24  size:1321760020 fileCount:10542 deletedBytes:1133223354 deletion:8312
collection:"records"    volumeCount:950 size:2170873344 fileCount:18456 deletedBytes:1945945038 deletion:15876
collection:"cpaas-rdbms-backup" volumeCount:264 size:61887771088        fileCount:23427 deletedBytes:3024085443 deletion:1347
collection:""   volumeCount:105 size:50902170504        fileCount:543283        deletedBytes:0  deletion:0
collection:"sfas-recs"  volumeCount:72  size:18427320108        fileCount:205613        deletedBytes:0  deletion:0
collection:"cpaas-app-tmp"      volumeCount:128 size:33076279088        fileCount:267203        deletedBytes:0  deletion:0
collection:"ml-models"  volumeCount:24  size:4729732432 fileCount:1804  deletedBytes:0  deletion:0
Total 7 collections.
>
> fs.cd buckets
> fs.ls
cache
cpaas-app-tmp
cpaas-rdbms-backup
ml-models
records
sfas-recs
>

What is this?
[
(collection:""   volumeCount:105 size:50902170504        fileCount:543283        deletedBytes:0  deletion:0)
]

**Comment by kmlebedev** [comment]
this is journal logs

**Reply by olegans1972** [reply]
And where is all this stored?

**Reply by kmlebedev** [reply]
> And where is all this stored?

```
weed shell
> fs.ls -a /topics/.system/log/
2024-07-18
2024-07-19
2024-07-20
2024-07-21
```

**Reply by olegans1972** [reply]
Thank you! Is it possible to disable them?In my case they take up too much space. Or set their storage period

**Comment by olegans1972** [comment]
I would like to send all logs only to stdout. Ноw I can do it?

**Reply by kmlebedev** [reply]
> I would like to send all logs only to stdout. Ноw I can do it?

This is a "transaction" log, why do you need it in stdout?

**Reply by olegans1972** [reply]
My seaweedfs works at the k8s env. 

```
[sifox@sifox-k3s-1 sifox_opodguyko_acc]$ k -n cpaas get po | grep seawe
seaweedfs-master-0                 1/1     Running     1 (12d ago)     12d
seaweedfs-master-1                 1/1     Running     1 (12d ago)     12d
seaweedfs-master-2                 1/1     Running     2 (12d ago)     12d
seaweedfs-sifox-filer-0            1/1     Running     0               94m
seaweedfs-sifox-filer-1            1/1     Running     0               94m
seaweedfs-sifox-filer-2            1/1     Running     0               95m
seaweedfs-volume-0                 1/1     Running     0               12d
seaweedfs-volume-1                 1/1     Running     0               12d
seaweedfs-volume-2                 1/1     Running     0               12d
seaweedfs-volume-3                 1/1     Running     0               12d
[sifox@sifox-k3s-1 sifox_opodguyko_acc]$
```

So sending logs to stdout is a default "kubernetes way" for any logs. We have the special service like Victoria logs which collects all logs from any pods to special database. 

 And now all transaction logs  store to the volume-server which is defined for another useful traffic.

**Reply by kmlebedev** [reply]
use
```
/usr/bin/weed -v  -logtostderr=true
```
